app/controllers/admin/todolist_controller.rb

```diff
@@ -1,6 +1,10 @@
 class Admin::TodolistController < Admin::AdminController
-
   def index
-    @petitions = Petition.for_state(Petition::SPONSORED_STATE).order(:created_at).paginate(:page => params[:page], :per_page => params[:per_page] || 20)
+    @petitions = Petition.in_moderation.order(:created_at)
+    if params[:petition] == 'collecting_sponsors'
+      @petitions = Petition.collecting_sponsors.order(:created_at)
+    end
+    @petitions = @petitions.paginate(page: params[:page], per_page: params[:per_page] || 20)
```

Comment:
Yes, much cleaner, thanks! Changes and removing redundant instant variables here:https://github.com/alphagov/e-petitions/commit/f8cde133388029058ff8bafe0cf7d34ef305a8a1